src/relay/transforms/fold_constant.cc

```diff
@@ -151,9 +151,12 @@ class ConstantFolder : public MixedModeMutator {
     }
 
     // We should think about potentially constant evaluation over these ops too.
-    if (call->op == invoke_tvm_op_ || call->op == shape_func_op_ || call->op == alloc_tensor_op_ ||
-        call->op == alloc_storage_op_ || call->op == device_copy_op_) {
```

Comment:
Do we need to add all these ops as non-computational? It looks like with this change things like `shape_func_op_` will now be handled differently.

Reply:
These ops are already marked as TNonComputational when they are registered

Reply:
I missed device_copy initially. It seems that marking device_copy as TNonComputational causes some problems in the fuse_ops pass, so I am leaving the explicit check for device_copy